Source/ThirdParty/o3dgc.js

```diff
@@ -181,16 +181,27 @@ define(['../Core/getTimeStamp'], function(getTimeStamp) {
         }
         return pos;
     }
+
     // Timer class
+    var windowPerformance = (typeof performance !== 'undefined') ? performance : {};
+    if (!windowPerformance.now) {
+        local.nowOffset = Date.now();
+        if (windowPerformance.timing && windowPerformance.timing.navigationStart) {
+            local.nowOffset = windowPerformance.timing.navigationStart;
+        }
+        windowPerformance.now = function now() {
```

Comment:
The problem here is that if the global `performance` object exists but `performance.now` does not exist, we end up creating the function `performance.now` on the global object, which means Cesium is polluting the global namespace.

@shunter do you agree?